client/stylesheets/_modal.scss

```diff
@@ -233,14 +233,55 @@
 	END END END END END END
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
 
+/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+	History Details Modal
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
+
+/*~~~~ Button ~~~~*/
 
+/*~~ Button End ~~*/
+
+#profileHistoryFeedback {
+	width: 100% !important;
+    background-color: #f8504b !important;
+    padding: 15px 0 !important;
+    text-align: center !important;
```

Comment:
!important should not be used<br>Line should be indented 2 spaces, but was indented 4 spaces